src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/UndeliverableListReportITest.java

```diff
@@ -0,0 +1,188 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/UndeliverableListReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+public class UndeliverableListReportITest extends AbstractStandardReportControllerITest {
```

Comment:
It will be worth adding this class name to the allowable value list in the StandardReportRequest :)

Reply:
Added